provenance.jq

```diff
@@ -0,0 +1,80 @@
+# input: "build" object (with "buildId" top level key)
+# output: array of image tags
+def tags:
+  .source.arches[].tags[],
+  .source.arches[].archTags[],
+  .build.img
+;
+
+# input: "build" object (with "buildId" top level key)
+# output: purl platform query string
+def platform_string:
+  .source.arches[].platformString | gsub("/"; "%2F")
+;
```

Comment:
that would be good! If so, we can remove this entirely from here. @LaurentGoderre, lets see if we can use the `uri` filter for the platform output. We just need to make sure that the substitution is there since PURL uses `%2F` not `/` in the string.

Reply:
@LaurentGoderre was able to use `@uri` outside of `provenance.jq` for `platform_string` so I removed this function entirely